rules/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector.php

```diff
@@ -86,7 +152,11 @@ public function refactorWithScope(Node $node, Scope $scope): ?Node
             return null;
         }
 
-        $scalarReturnType = $this->strictScalarReturnTypeAnalyzer->matchAlwaysScalarReturnType($node, $scope);
+        $scalarReturnType = $this->strictScalarReturnTypeAnalyzer->matchAlwaysScalarReturnType(
```

Comment:
In case the `Scope $scoped` argument is dropped from `matchAlwaysScalarReturnType` this should be updated as well:
```suggestion
        $scalarReturnType = $this->strictScalarReturnTypeAnalyzer->matchAlwaysScalarReturnType($node, $this->hardCodedOnly);
```